Dockerfile

```diff
@@ -1,74 +1,139 @@
-# Copyright (c) 2020 Red Hat, Inc.
+# Copyright (c) 2021 Red Hat, Inc.
 # This program and the accompanying materials are made
 # available under the terms of the Eclipse Public License 2.0
 # which is available at https://www.eclipse.org/legal/epl-2.0/
 #
 # SPDX-License-Identifier: EPL-2.0
 #
 
-FROM fedora:32
-
-# Product name:
-#   ideaIC    - IntelliJ Idea Community
-#   ideaIU    - IntelliJ Idea Ultimate
-#   WebStorm  - WebStorm
-ARG PRODUCT_NAME="ideaIC"
-
-# Product version
-ARG PRODUCT_VERSION="2020.2.3"
-
-# Product url constructs based on PRODUCT_NAME and PRODUCT_VERSION:
-#   https://download.jetbrains.com/idea/ideaIC-2020.2.3.tar.gz
-#   https://download.jetbrains.com/idea/ideaIU-2020.2.3.tar.gz
-#   https://download.jetbrains.com/webstorm/WebStorm-2020.2.3.tar.gz
-
-ARG BASE_MOUNT_FOLDER="/JetBrains"
+#
+# Copyright 2019-2020 JetBrains s.r.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
 
-# which is used by novnc to find websockify
-RUN yum install -y tigervnc-server supervisor wget java-11-openjdk-devel novnc fluxbox git which nss libXScrnSaver mesa-libgbm
+# Stage 1. Download IDE by downloadUrl build argument
+#   Requires only wget to download the IDE distributive.
+FROM alpine:latest as ideDownloader
```

Comment:
please use a specific tag like 3.12.3, etc.

Reply:
Used 3.13.4 tag in both cases